Cargo.toml

```diff
@@ -6,18 +6,18 @@ edition = "2018"
 
 [dependencies]
 chrono = "0.4"
-futures-lite = "1.10"
-egui = "0.2"
-egui_wgpu_backend = "0.1"
-egui_winit_platform = "0.1"
+futures-lite = "1.11"
+egui = "0.3"
+egui_wgpu_backend = { version = "0.2", git = "https://github.com/hasenbanck/egui_wgpu_backend" }
```

Comment:
Remove the override here and localy use the [patch.crates-io] or the crate specific patch please.

Reply:
Fixed. I was confused before because I didn't understand how the `patch` property worked, but it should be correct now